client/components/QuestionsAndAnswers/AnswerItem.jsx

```diff
@@ -18,34 +18,33 @@ const AnswerItem = ({ answer }) => (
         {' '}
```

Comment:
One comment I have is for how you're spacing out items. It looks like you're using spaces with `{' '}` to space out items, however, this can be something that is solved with padding & would be more adjustable with css.